pkg/sql/sem/builtins/builtins.go

```diff
@@ -3685,7 +3685,39 @@ may increase either contention or retry errors, or both.`,
 			},
 			Info: "This function is used only by CockroachDB's developers for testing purposes.",
 		},
-	),
+		tree.Overload{
+			Types:      tree.ArgTypes{{"val", types.Geography}},
+			ReturnType: tree.FixedReturnType(types.Int),
+			Fn: func(ctx *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
+				arg := args[0]
+				if arg == tree.DNull {
+					return tree.DZero, nil
+				}
+				// TODO: change this call. We need the index configuration to know
+				// how many entries there will be. How do we get that?
+				arr := tree.MustBeDGeography(arg)
```

Comment:
I think rather than try to figure that out, you should make a new builtin and change the caller in `validateInvertedIndexes`. That thing will have all of the context, whereas plumbing it into the builtins will be extremely awkward.